src/main.js

```diff
@@ -1,19 +1,25 @@
-// Create variables targetting the relevant DOM elements here 👇
+var updatedCoverImage = document.querySelector('.cover-image');
+var updatedTitle = document.querySelector('.cover-title');
+var updatedDescriptorOne = document.querySelector('.tagline-1');
+var updatedDescriptorTwo = document.querySelector('.tagline-2');
```

Comment:
Good global variable names, make a mental note to come back to these further on in the project and see if they need to be global, or maybe they can be just local.